doc/docker.md

```diff
@@ -0,0 +1,64 @@
+[<- Back to Index](index.md)
+
+# Docker Image
+
+Yet Analytics publishes Docker container images of SQL LRS on [DockerHub](https://hub.docker.com/repository/docker/yetanalytics/lrsql) in the format `yetanalytics/lrsql:<release version | latest>`.
+
+## Usage
+
+You can run SQL LRS directly from the Docker CLI in its default SQLite configuration:
+
+``` shell
+docker run \
+    -it \
+    -p 8080:8080 \
+    -e LRSQL_API_KEY_DEFAULT=my_key \
+    -e LRSQL_API_SECRET_DEFAULT=my_secret \
+    -e LRSQL_ADMIN_USER_DEFAULT=my_username \
+    -e LRSQL_ADMIN_PASS_DEFAULT=my_password \
+    yetanalytics/lrsql:latest
+```
```

Comment:
Maybe add a comment saying "You can then communicate with the server, in this example at `0.0.0.0:8080`? I know it's not necessary for someone who knows what they're doing, but I'm speaking as someone who doesn't know what they're doing.

Reply:
done. I think in general this is intended to be for more advanced users, though it does occur to me that docker is hands-down the _quickest_ way for someone to "try out" lrsql, is it maybe worth linking to this section from getting started?